images/cowork.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 125" x="0px" y="0px"><title>Artboard 60</title><path d="M89,54H82.93a2.46,2.46,0,0,0,.07-.5A2.5,2.5,0,0,0,81,51s0,0,0,0V38a1,1,0,0,0-1-1H60a1,1,0,0,0-1,1V51s0,0,0,0a2.47,2.47,0,0,0-2,3H48V50.29A7.3,7.3,0,0,0,40.71,43H29.29A7.3,7.3,0,0,0,22,50.29V54H21v0c-1-2.22-2.3-3.95-4.46-4.29L12,36.17,26.75,18.08l.28.63a.91.91,0,0,0,.09.15,6.93,6.93,0,0,0-.85,7.39l2,4.09a1,1,0,0,0,1.34.45l10.76-5.31a1,1,0,0,0,.45-1.34l-2-4.09a6.94,6.94,0,0,0-6.4-3.86.94.94,0,0,0,0-.14l-1.77-3.59A1,1,0,0,0,29.28,12l-.9.44h0l-.44-.9a1,1,0,0,0-1.79.88l.44.9h0l-.9.44a1,1,0,0,0-.45,1.34l.56,1.14a1,1,0,0,0-.44.29L9,36.49a1,1,0,0,0,.77,1.63,1,1,0,0,0,.54-.19l3.91,11.87a5,5,0,0,0-3.72,4.06,1,1,0,0,0,0,.26A1,1,0,0,0,10,55V89a1,1,0,0,0,2,0V60H22.79a7.29,7.29,0,0,0,6.5,4H34v5H26a4,4,0,0,0,0,8h8v7.35l-8.41,3.74a1,1,0,0,0,.81,1.83L35,86.09l8.59,3.82a1,1,0,0,0,.81-1.83L36,84.35V77h8a4,4,0,0,0,0-8H36V64h4.71a7.29,7.29,0,0,0,6.5-4H88V89a1,1,0,0,0,2,0V55A1,1,0,0,0,89,54ZM37,20.94l1.58,3.2-9,4.42-1.58-3.2a5.07,5.07,0,0,1,2.36-6.67A4.94,4.94,0,0,1,37,20.94ZM30.4,16.55a6.92,6.92,0,0,0-.87.35,7.17,7.17,0,0,0-.91.55l-1.14-2.32,1.79-.88ZM46,73a2,2,0,0,1-2,2H26a2,2,0,0,1,0-4H44A2,2,0,0,1,46,73ZM61,39H79V51H61ZM59.32,54a.5.5,0,0,1,.18-1h21a.5.5,0,0,1,.19,1l-.06,0H59.37ZM15.55,51.62a3.09,3.09,0,0,1,3,2.38h-6A3.06,3.06,0,0,1,15.55,51.62ZM12,58V56H22v.71A7.28,7.28,0,0,0,22.12,58Zm28.71,4H29.29A5.3,5.3,0,0,1,24,56.71V50.29A5.3,5.3,0,0,1,29.29,45H40.71A5.3,5.3,0,0,1,46,50.29v6.42A5.3,5.3,0,0,1,40.71,62ZM48,56.71V56H88v2H47.88A7.28,7.28,0,0,0,48,56.71Z"/><path d="M80,65H62a4,4,0,0,0,0,8h8V84.35l-8.41,3.74a1,1,0,0,0,.81,1.83L71,86.09l8.59,3.82a1,1,0,1,0,.81-1.83L72,84.35V73h8a4,4,0,0,0,0-8Zm0,6H62a2,2,0,0,1,0-4H80a2,2,0,0,1,0,4Z"/><path d="M67,41.85a1,1,0,0,0-1.41-1.41l-3.42,3.42a1,1,0,1,0,1.41,1.41Z"/><path d="M69.29,40.56l-5.87,5.88a1,1,0,1,0,1.41,1.41L70.71,42a1,1,0,0,0-1.41-1.41Z"/><path d="M89,10H46a1,1,0,0,0-1,1V31a1,1,0,0,0,1,1H89a1,1,0,0,0,1-1V11A1,1,0,0,0,89,10ZM88,30H47V12H88Z"/><path d="M50,29h9a1,1,0,0,0,1-1V14a1,1,0,0,0-1-1H50a1,1,0,0,0-1,1V28A1,1,0,0,0,50,29Zm1-14h7V27H51Z"/><path d="M65,20h7a1,1,0,0,0,1-1V14a1,1,0,0,0-1-1H65a1,1,0,0,0-1,1v5A1,1,0,0,0,65,20Zm1-5h5v3H66Z"/><path d="M77,20h7a1,1,0,0,0,1-1V14a1,1,0,0,0-1-1H77a1,1,0,0,0-1,1v5A1,1,0,0,0,77,20Zm1-5h5v3H78Z"/><path d="M65,29h7a1,1,0,0,0,1-1V23a1,1,0,0,0-1-1H65a1,1,0,0,0-1,1v5A1,1,0,0,0,65,29Zm1-5h5v3H66Z"/><path d="M77,29h7a1,1,0,0,0,1-1V23a1,1,0,0,0-1-1H77a1,1,0,0,0-1,1v5A1,1,0,0,0,77,29Zm1-5h5v3H78Z"/><path d="M56,17H53a1,1,0,0,0,0,2h3a1,1,0,0,0,0-2Z"/><path d="M56,20H53a1,1,0,0,0,0,2h3a1,1,0,0,0,0-2Z"/><text x="0" y="115" fill="#000000" font-size="5px" font-weight="bold" font-family="'Helvetica Neue', Helvetica, Arial-Unicode, Arial, Sans-serif">Created by Maria Kislitsina</text><text x="0" y="120" fill="#000000" font-size="5px" font-weight="bold" font-family="'Helvetica Neue', Helvetica, Arial-Unicode, Arial, Sans-serif">from the Noun Project</text></svg>
```

Comment:
Do you have a worst image to integrate please? :trollface:

Reply:
yes